### ISSUE TITLE
[Accessibility]: CodeInsights Edit UI - Chart does not announce itself properly

### ISSUE THREAD

**Issue by unclejustin** [sourcegraph/sourcegraph#41687]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33501

### Problem description

![PixelSnap 2022-09-15 at 17 18 32](https://user-images.githubusercontent.com/1855233/190518598-0f43fd16-46a7-4a2d-a3a3-cddb826dae9d.png)

When the chart is focused it simply announces "Group. You are on a group"

### Expected behavior

It would announce that a "chart" is focused.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

/cc @joelkw @felixfbecker @vovakulikov
